samples/open-telemetry/logging/Core_8/GenericHost/MyMessageHandler.cs

```diff
@@ -0,0 +1,24 @@
+﻿using System.Threading.Tasks;
+using Microsoft.Extensions.Logging;
+using NServiceBus;
+
+public class MyMessageHandler : IHandleMessages<MyMessage>
+{
+    private readonly ILogger<MyMessageHandler> logger;
+
+    public MyMessageHandler(ILogger<MyMessageHandler> logger)
+    {
+        this.logger = logger;
+    }
+
+    public Task Handle(MyMessage message, IMessageHandlerContext context)
+    {
+        #region log-statement
+
+        logger.LogInformation($"Received message #{message.Number}");
```

Comment:
String interpolation should not be used when logging.

https://learn.microsoft.com/en-us/dotnet/fundamentals/code-analysis/quality-rules/ca2254

https://www.youtube.com/watch?v=6zoMd_FwSwQ

Reply:
agreed, should be using structured logging and unfortunately we are a serious violator: https://github.com/Particular/NServiceBus/issues/6136

Reply:
Applied your feedback in https://github.com/Particular/docs.particular.net/commit/d472f003fd6214942d5dc66e1d148bee9bf32d8b for this specific sample.